src/actions/exchangeActions.js

```diff
@@ -188,6 +188,7 @@ export const searchOffersAction = (fromAssetCode: string, toAssetCode: string, f
     } = getState();
 
     const activeWalletId = getPreferredWalletId(accounts);
+    const isSmartWallet = isSmartWalletActive(accounts);
```

Comment:
You can get the same without creating new util: `const isSmartWallet = isActiveAccountSmartWalletSelector(getState())`

Reply:
too much passing around

Reply:
This is what are selectors for